plans/e2e/tier-2.fmf

```diff
@@ -0,0 +1,38 @@
+summary: qm tier-2 test plan
+
+discover:
+    how: fmf
+    filter: tag:tier-2
+
+environment+:
+  CONTROL_CONTAINER_NAME: host
+  FFI_SETUP_OPTIONS: none
+  PACKAGE: qm
+  PACKIT_COPR_PROJECT: release
+  SET_CS_REPO: yes
+  SKIP_EPEL_REPO: yes
+
```

Comment:
SET_CS_REPO: yes
  SKIP_EPEL_REPO: yes
  
  Are not related to this repo

Reply:
removed